libraries/standard/http/include/http_client.h

```diff
@@ -403,19 +439,44 @@ HTTPStatus_t HTTPClient_AddHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * #HTTPRequestHeaders_t.pBuffer.
  *
  * For example, if requesting for the first 1kB of a file the following would be
- * written  "Range: bytes=0-1024\r\n".
+ * written  "Range: bytes=0-1024\r\n\r\n".
+ *
+ * The trailing "\r\n" that denotes the end of the header lines is overwritten, if it
+ * already exists in the buffer.
+ *
+ * @note There are 3 different forms of range specification, determined by the
+ * combination of @a rangeStartOrLastNBytes and @a rangeEnd parameter values:
+ * 1. Request containing both parameters for the byte range [rangeStart, rangeEnd]
+ * where @a rangeStartOrLastNBytes <= @a rangeEnd.
+ * Example request: "Range: bytes=0-1024\r\n" for requesting bytes in the range [0, 1024].
  *
- * TODO: Add documentation about rangeStart and rangeEnd configuration.
+ * 2. Request for the last N bytes, represented by @p rangeStartOrLastNBytes.
+ * @p rangeStartOrLastNBytes should be negative and @p rangeEnd should #FILE_END.
+ * Example request: "Range: bytes=-512\r\n" for requesting the last 512 bytes
+ * (or bytes in the range [0, 512]).
```

Comment:
The comment in the parentheses is not correct if the file is greater than 513 bytes.

Reply:
Good catch. Fixed